doc/en/broken-dep-constraints.txt

```diff
@@ -3,4 +3,4 @@
 
 # Pin towncrier temporarily due to incompatibility with sphinxcontrib-towncrier:
 # https://github.com/sphinx-contrib/sphinxcontrib-towncrier/issues/92
-towncrier!=24.7.0,!=24.7.1
+towncrier<24.7
```

Comment:
@Pierre-Sassoulas I wanted to revert the PR, but it contains more things than this line… FYI, the extension is now compatible with new Towncrier.

Reply:
There should be only 2 conflicts during the revert and we can keep the line with the higher value each time.

Reply:
Although it's probably better to start from scratch.

Reply:
Yeah, I was just being lazy in the middle of the night :) So I figured I'd at least leave a notification in here.